src/modules/twap/containers/TwapFormWidget/tooltips.tsx

```diff
@@ -0,0 +1,56 @@
+export interface LabelTooltipObject {
```

Comment:
no need to have the object suffix here imo

Reply:
Maybe just `TooltipInfo`?

Reply:
Fixed